File: 0.C_CPP_JAVA Programs/JAVA/Program16.java
/*1.Write a Java program which accept string from user and count number of capital characters.
Input :   “Marvellous Multi OS”
Output :   4

 */
import java.lang.*;
import java.util.*;
class DemoString
{
    int Count(String str)
    {
        char arr[] = str.toCharArray();
        int size=arr.length;
        int i=0;
        int iCount=0;
        while(i<size)
        {
            if(arr[i]>='A'&&arr[i]<='Z')
            {
                iCount++;
            }
            i++;
        }
        return iCount;
    }
}
class Demo
{
    public static void main(String arg[])
    {

         Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the string");
        String str=sobj.nextLine();

        DemoString dobj=new DemoString();

        int ret=dobj.Count(str);
        System.out.println("Capital letter is: " + ret);
    }
}

File: 1.Python/Program16.py
# 4.Write a program which accept one number form user and return addition
# of its factors.
# Input :  12
# Output : 16  (1+2+3+4+6)
def AddFactors(No):
     iCnt=1
     iSum=0
     for iCnt in range(1, No, 1):
        if No % iCnt == 0:
           iSum=iSum+iCnt
     return iSum

def Main():

   No=0
   iRet=0

   print("Enter the one number:")
   No =int(input())

   iRet=AddFactors(No)
   print("Your addition of factors is: ", iRet)

if __name__ == "__main__":
   Main()